src/app/modules/timesheet/components/timesheet-edit/timesheet-edit.component.ts

```diff
@@ -7,23 +7,22 @@ import * as _ from 'lodash';
 
 import { ReviewMail } from 'src/app/shared/models/review-mail.model';
 
-import { CalendarService } from 'src/app/modules/calendar/calendar.service';
+import { Timesheet } from 'src/app/shared/models/timesheet.model';
+import { Invoice } from 'src/app/shared/models/invoice.model';
```

Comment:
think about using :

`import { ReviewMail, Timesheet, Invoice } from 'src/app/shared/models';`

When you can.

PS: To do that you will need an index in the `models` folder if you don't already have it